pkgs/games/zdoom/bcc-git.nix

```diff
@@ -1,7 +1,8 @@
 { lib, stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation {
-  name = "doom-bcc-git-0.8.0.2018.01.04";
+  pname = "doom-bcc-git";
+  version = "0.8.0.2018.01.04";
```

Comment:
```suggestion
  pname = "doom-bcc";
  version = "unstable-2018-01-04";
```

Reply:
could you explain when i should use `unstable`? and why is this not an optional flag?